app/client/views/front/front.js

```diff
@@ -1,7 +1,28 @@
 Template.front.helpers({
-  props(){
-    return { 
-      Modal
-    }
+  props() {
+    return {
+      Modal,
+      localizedText: {
+        "quickAddTitle": TAPi18n.__("front-quickAddPanel-title"),
```

Comment:
I think the approach here is a bit cumbersome, in that we have so much 'wiring' to do to get the localization strings into the template. Lets try a different approach:

1. make sure we can import `TAPi18n` inside of a Vue component, e.g. in the `created` lifecycle hook
2. define a global [filter](https://vuejs.org/v2/guide/filters.html) such as `localize` that takes a localization `key` as an argument and returns the localized string based on the current locale
    - this `localize` filter will basically wrap `TAPi18n.__("localizationKey")`
3. use the filter in the Vue templates
    - e.g. `{{ "localizationKey" | localize }}`

There may be a better approach than [Vue filters](https://vuejs.org/v2/guide/filters.html), but they seem to be the closest to the Handlebars/TAPi18n syntax of `{{_ "localizationKey" }}`.

Reply:
With the above solution, all that will be needed is to pass the `locale` into the component in a reactive context, such as a `prop`.